src/omega_edit/client.ts

```diff
@@ -137,7 +137,8 @@ export function activate(ctx: vscode.ExtensionContext) {
             message.newFile ? message.newFile : false,
             fileSize,
             searchPattern,
-            message.replaceText ? message.replaceText : ''
+            message.replaceText ? message.replaceText : '',
+            message.caseSensitive ? message.caseSensitive : false
```

Comment:
This should be message.case**In**sensitive, and false is the correct default value.